relayer/utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+def get_elapsed_time_in_milliseconds(start_time, end_time):
+    elapsed_time = end_time - start_time
+    return elapsed_time.microseconds / 1000.0 + elapsed_time.seconds * 1000
```

Comment:
I really don't get this formula, I feel there is double time.
What does `some.microseconds` and `some.seconds` return?